corgidrp/recipe_templates/l1_to_l2b.json

```diff
@@ -9,6 +9,9 @@
     "steps" : [
         {
             "name" : "prescan_biassub",
+            "calibs" : {
+                "DetectorNoiseMaps" : "AUTOMATIC"
```

Comment:
why isn't this one OPTIONAL, but it is in `l1_to_l2a_eng.json` and `l1_to_l2a_basic.json`?